### ISSUE TITLE
Better error handling / descriptions

### ISSUE THREAD

**Issue by schumannd** [oblador/react-native-collapsible#156]
First of all, great plugin!

A minor issue I had while debugging my usage of it:

I ran into two errors which were not very helpful:
> Error: Invariant Violation: Touchable child must either be native or forward setNativeProps to a native component stack

and 

> React.Children.only expected to receive a single React element child

Both were some result of providing invalid values to `sections`, `renderHeader` and `renderContent`.

**Comment by iRoachie** [comment]
Hey! The second error you saw:
> React.Children.only expected to receive a single React element child

Is a propTypes error from React. It throws that when you're trying to pass in more than one element to that function.

E.g.
```jsx
renderContent={() => (
  <Text>Hey</Text>
  <Text>Schumand</Text>
)}
```

Instead you need to create a wrapper element since the function only accepts one element:

```jsx
renderContent={() => (
  <View>
    <Text>Hey</Text>
    <Text>Schumand</Text>
  </View>
)}
```

For: 
> Error: Invariant Violation: Touchable child must either be native or forward setNativeProps to a native component stack

What code did you write that produced this error?

**Comment by schumannd** [comment]
That react error was the result of passing a function, which does not return any react child.
It is a dumb mistake, but it happens easily when juggling multiple nested maps.

The other one I do not remember. Maybe passing `undefined` for `sections`?

**Comment by iRoachie** [comment]
Thanks @schumannd will definitely add in an error check

**Comment by iRoachie** [comment]
Added in https://github.com/oblador/react-native-collapsible/commit/df781b389293d11ae2211e628ddc3f386eff1f9f.

![simulator screen shot - iphone 6 - 2018-03-17 at 23 08 07](https://user-images.githubusercontent.com/5962998/37562198-ca042218-2a38-11e8-8a95-950fdcc3f478.png)

**Comment by iRoachie** [comment]
Released in [v0.11.0](https://github.com/oblador/react-native-collapsible/releases/tag/v0.11.0)
